### DISCUSSION TITLE
Can you please release v5.1.2 of @prisma/instrumentation so the cves go away?

### DISCUSSION THREAD

**Discussion by toddtarsi** [prisma/prisma#20737] [Q&A] [answered]
### Bug description

This would get the mainline version out of trouble with the import-in-the-middle stuff. Thanks for the awesome codebase!

https://github.com/advisories/GHSA-f8pq-3926-8gx5

### How to reproduce

---

### Expected behavior

---

### Prisma information

---

### Environment & setup

---

### Prisma Version

---

**Comment by SevInf** [comment, marked as answer]
Hi @toddtarsi.
Prisma 5.2 is scheduled for release for Tuesday, August 22d and it will include the update you mentioned. If you want to have something earlier, you can try dev snapshot, for example `5.2.0-dev.64`. Normally, we don't recommend using dev snapshots in production, however, if you are just bumping `@prisma/instrumentation` package only, nothing changed in there between 5.1 and 5.2 except the dependency update you want, so it should be safe. 

Let me know if this works for you.

**Reply by toddtarsi** [reply]
@SevInf - That absolutely does. Thank you for explaining all of this to me!

**Reply by SevInf** [reply]
Glad I could help! One correction: I typed in wrong date for 5.2 release initially, it happens next week, on 22d of August, not 29th.
